### PR TITLE
Update UFS_UTILS for RRFSFW, remove unneeded modules from run.ver

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 

- This PR contains two main updates (see below):  
- The UFS_UTILS code is updated for the RRFS fire weather nest to include the chgres_cube code changes needed to support rotated lat-lon grids with the WMO template.  This is what RRFS uses, and the fire weather grid is initialized using the RRFS rotated lat-lon data.  With this update, I've also changed RRFSFW to use the native level (natlev) files for ICs/LBCs instead of the pressure level (prslev) files.  The make_orog ex-script had to be updated to work with the newer version of UFS_UTILS, and some RRFS settings in the make_ics ex-script were modified to not use climatological values.
- The run.ver (and build.ver files) were updated to remove unneeded modules.  I looked at build.ver and run.ver of AQM as a starting point.  These changes are needed for EE2 compliance.  When running specific tasks, NCO only wants the modules loaded that are needed at runtime.  I tested all the jobs in the non-DA workflow with RRFSFW, and I also ran part of a DA engineering test to make sure the processing and analysis jobs would complete successfully.  I did not test the run_bufrsnd task, but the only change there is removing the PrgEnv-Intel module and that one was not needed by any other tasks.  Perhaps these module changes should also be tested in the v0.8.5 real-time runs.

### Machines/Platforms:

- WCOSS2
  - [x] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [x] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [x] Parallel
- [x] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## CONTRIBUTORS (optional): 

@LarissaReames-NOAA and @GeorgeGayno-NOAA assisted with the UFS_UTILS code changes.

### CODE REVIEW

File: sorc/Externals.cfg [hunk 1]
```diff
@@ -2,8 +2,8 @@
 protocol = git
 repo_url = https://github.com/ufs-community/UFS_UTILS
```
Comment (GeorgeGayno-NOAA) [inline sorc/Externals.cfg:3]: I am going to create a tag for you. Don't use the release branch.

Comment (BenjaminBlake-NOAA) [reply sorc/Externals.cfg:3]: Sounds good, I can update that.

Comment (GeorgeGayno-NOAA) [reply sorc/Externals.cfg:3]: Please use this tag: https://github.com/ufs-community/UFS_UTILS/releases/tag/ufs_utils_1_12_2

I am going to delete the release branch. If you need further updates, I can create a new release branch from the tag.

Comment (MatthewPyle-NOAA) [reply sorc/Externals.cfg:3]: Thanks @GeorgeGayno-NOAA for being willing to create a fresh release branch if it becomes needed.

**Comment by BenjaminBlake-NOAA** [comment]
@MatthewPyle-NOAA I'm converting this PR back to a draft until I'm able to update the UFS_UTILS checkout.  I also saw Rahul posted in the kickoff meeting slides that w3nco has been deprecated and should be removed, so I'll need to remove that from build.ver and make sure the codes still build without it.

**Review by MatthewPyle-NOAA — DISMISSED** [review]
Looks ready to test to me - one small question about the run.ver changes.

File: versions/run.ver
[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply versions/run.ver:1]: @MatthewPyle-NOAA  Not that I'm aware of.  It should be safe to remove gsmgfs_ver, hrrr_ver, nam_ver, and rap_ver from run.ver.  The HRRR is of course used in the ensemble post-processing but that is a separate workflow.

**Review by MatthewPyle-NOAA — APPROVED** [review]
@BenjaminBlake-NOAA Hopefully removing those model versions don't break anything, but thanks!

**Pull request merged**